_data/templates.json

```diff
@@ -1301,6 +1301,16 @@
 				"language": "fr",
 				"path": "topic-fr.html"
 			},
+			{
+				"title": "[Topic title without stacked header]",
+				"language": "en",
+				"path": "topic-nostackedheader-en.html"
+			},
+			{
+				"title": "[Titre du sujet sans en-tête superposée]",
```

Comment:
Can you please remove the extra "e" at the end of the following sentence: "Titre du sujet sans en-tête superposée"